### PR TITLE
D8NID-282 Add custom module to control display of sidebar block view.

### PR DESCRIPTION
**Pull request by johangant**
This PR provides a module that gives a block to show the current position in the site themes taxonomy vocab, based on values from the node being viewed.

### CODE REVIEW

File: nidirect_related_content/nidirect_related_content.module [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+/**
+ * Implements hook_views_pre_build().
+ */
+function nidirect_related_content_views_pre_build(ViewExecutable $view) {
+  if ($view->id() != 'articles_by_term') {
+    return;
+  }
+
+  if ($view->current_display != 'block_1') {
```
Comment (omahm) [inline nidirect_related_content/nidirect_related_content.module]: I'd be tempted to put this in as an && condition to the if statement above as a blanket condition for the correct View and Display.

File: nidirect_related_content/nidirect_related_content.module [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+/**
+ * Implements hook_views_pre_render().
+ */
+function nidirect_related_content_views_pre_render(ViewExecutable $view) {
+  if ($view->id() != 'articles_by_term') {
+    return;
+  }
+
+  if ($view->current_display != 'block_1') {
```
Comment (omahm) [inline nidirect_related_content/nidirect_related_content.module]: As per previous comment.

File: nidirect_related_content/nidirect_related_content.module [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+/**
+ * Implements hook_preprocess_views_view_field().
+ */
+function nidirect_related_content_preprocess_views_view_field(&$variables) {
+  if ($variables['view']->id() != 'articles_by_term') {
+    return;
+  }
+
+  if ($variables['view']->current_display != 'block_1') {
```
Comment (omahm) [inline nidirect_related_content/nidirect_related_content.module]: as per previous comment.

File: nidirect_related_content/nidirect_related_content.module (hunk 1 above)
Comment (johangant) [reply nidirect_related_content/nidirect_related_content.module]: Probably more concise, agreed. I'll crunch them down a bit as suggested 👍

**Pull request merged**